### ISSUE TITLE
Activity viewer used in Design Studio 1.6 SP1

### ISSUE THREAD

**Issue by jainjyo** [org-scn-design-studio-community/sdkpackage#156]
Team,

Can i use this component and bind it to SAP BW as data source.

**Comment by marloh65** [comment]
Yes you can. I did this already based on a SAP BW 7.4 Query, based on a composite provider as the primary data source. Just needs a little scripting for read out.
